test/data/adult-test.txt

```diff
@@ -0,0 +1,119 @@
+#@ TextLoader{
```

Comment:
May want to remove the ML.NET header.

Also, we'll want to note in our dataset README.md that we're redistributing a modified version of the Adult dataset. I'm not sure if it's otherwise modified, but this is at least a subsample of the rows.